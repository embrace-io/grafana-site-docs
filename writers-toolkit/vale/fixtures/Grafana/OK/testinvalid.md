Don't use:

- O.K.
- OK
- ok
- Ok
- okay
- Okay

Also avoid:

- A-OK
- hokay
- k
- keh
- kk
- M'kay
- oka
- okeh
- Okie dokie
- Okily Dokily
