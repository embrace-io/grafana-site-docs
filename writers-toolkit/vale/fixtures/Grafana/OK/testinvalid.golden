testinvalid.md:4:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:5:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:6:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:7:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:8:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:12:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:13:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:14:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:15:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:16:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:17:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:18:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:19:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:20:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
testinvalid.md:21:3:Grafana.OK:Don't use any variation of okay in prose. The exceptions are when you’re referencing or quoting: - A user interface - HTTP status codes or other code 
